1.hello-world/README.md

```diff
@@ -94,7 +94,9 @@ This calls out to our Redis cache to grab the latest value of the "order" key, w
 
 1. Install dependencies: `npm install`. This will install `express` and `body-parser`, dependencies that are shown in our `package.json`.
 
-2. Run Node.js app with Dapr: `dapr run --app-id mynode --app-port 3000 --port 3500 node app.js`. This should output text that looks like the following, along with logs:
+2. Run Node.js app with Dapr: `dapr run --app-id mynode --app-port 3000 --port 3500 node app.js`. **Note**: the port the app runs on is configurable and can be determined by the developer. Our Node app happens to run on port 3000. 
```

Comment:
Can we use `>` markdown keyword? like below.

> Note : ....

Reply:
Yep! Will do.

Reply:
@youngbupark I've updated the change to use the markdown keyword and combined it with the other "Note" we already had about the run command